apiserver/environment.yml

```diff
@@ -17,7 +17,5 @@ dependencies:
     - decorator==4.3.0
     - aiohttp
     - aiodns
-    - google-api-python-client
-    - google.cloud.storage
-    - jgscm==0.1.*
+    - https://github.com/hail-is/jgscm/archive/master.zip
```

Comment:
When we have a stable version, we can pin to a specific commit in a similar way

`https://github.com/hail-is/jgscm/archive/hash_of_commit.zip`